### ISSUE TITLE
Persona image doesn't support initial names in other languages

### ISSUE THREAD

**Issue by saraXiangL** [OfficeDev/office-ui-fabric-react#9838]
**Describe the feature that you would like added**

In Persona, when we generate the icon based on initial name in other language, like 森雪株式会社, it displays blank.

Can Persona support initial name in other languages instead of displaying blank? It's available in a similar component react-avatar (https://www.sitebase.be/react-avatar/)

**What component or utility would this be added to**

Persona

**Have you discussed this feature with our team, and if so, who**

Confirmed it with Micah Godbolt

**Additional context/screenshots**

**Comment by micahgodbolt** [comment]
Looking at the react-avatar examples passing in the (name?) of  森雪株式会社 returns a persona with the first character in the circle 

![image](https://user-images.githubusercontent.com/1434956/61400758-ccb45d80-a885-11e9-8514-372dfecbe842.png)

If this is an acceptable 'initial' this should be pretty easy to implement.

**Comment by micahgodbolt** [comment]
@saraXiangL Do you know if using the first character is the preferred initials in this case? @betrue-final-final, any idea if there is precedent here?

**Comment by saraXiangL** [comment]
> @saraXiangL Do you know if using the first character is the preferred initials in this case? @betrue-final-final, any idea if there is precedent here?

Hi Micah. Thank you for your reply. The first character in other languages might have a totally different meaning. In my opinion, a default gender neutral user icon is preferred in this situation.

**Comment by micahgodbolt** [comment]
Ah, so you aren't proposing us try to infer the symbol to use, but rather have a fallback generic icon be rendered if we cant infer the initials

TLDR: use contact icon if initials can't be infered
![image](https://user-images.githubusercontent.com/1434956/61418203-a0f99d80-a8ae-11e9-8433-aa87aada7fdc.png)

**Comment by saraXiangL** [comment]
Yeah... After discussion, we think this should be a better and easier way.

**Comment by joschect** [comment]
@saraXiangL  and @micahgodbolt sounds good. I'll get this into pr!

**Comment by joschect** [comment]
Actually, @saraXiangL  and @micahgodbolt after looking at this, I'm confused as to why it's showing us blank rather than showing the icon. Looking at the code it appears that's what we already do, and we already have some test cases in the examples. 

@saraXiangL where are you seeing them show up blank? There might be a bug somewhere if you can share a repro.

![image](https://user-images.githubusercontent.com/20801176/61551475-2180e100-aa0a-11e9-8d27-33c7aa1946b9.png)

**Comment by micahgodbolt** [comment]
Hah....at least we proposed a solution that others came to as well :D

**Comment by saraXiangL** [comment]
Thank you for your reply! Here is the screenshot that I get the blank icon. Here I used props.text, which is "森雪株式会社"， and props.imageUrl, which is an empty string for this company. I'm wondering if that is why it couldn't display the default icon? 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/52979394/61554844-c43d5d80-aa12-11e9-83e6-e186c458fa3f.png)

**Comment by joschect** [comment]
I'm still not able to reproduce that. What version of fabric are you using? Maybe this was a bug in an old one that got fixed?

If you're already on a recent version, would you mind using this [codepen](https://codepen.io/slimemold/pen/aezMzO) to try to repro it? It's in the state, that I think is what you mentioned.

**Comment by saraXiangL** [comment]
Thank you so much for your time. I just double checked it. Yeah, this was a bug in version 3.8. In our new build with the latest version, it's fixed.
